frontend/app/templates/components/transportation/study-area-map.hbs

```diff
@@ -23,24 +23,51 @@
         paint=(hash fill-opacity=0)
       }} as |layer|
     >
-      <Mapbox::LayerHoverCard
+      <Mapbox::CurrentMouseoverFeatures
         @map={{map}}
-        @layerId={{layer.layerId}} as |hover-card-data|
+        @layerId={{layer.layerId}}
+        as |mouseover|
       >
-        <Transportation::StudyAreaMap::CensusTractPopup
-          @feature={{hover-card-data}}
-          data-test-popup='census-tract'
-        />
-      </Mapbox::LayerHoverCard>
+          {{#if (and mouseover.point mouseover.features)}}
+            <HoverCard @point={{mouseover.point}}>
+              <Transportation::StudyAreaMap::CensusTractPopup
+                @feature={{mouseover.features.firstObject}}
+                data-test-popup='census-tract'
+              />
+            </HoverCard>
+          {{/if}}
+      </Mapbox::CurrentMouseoverFeatures>
       <Mapbox::FeatureSelector
         @map={{map}}
         @layerId={{layer.layerId}} as |selectedFeatures| >
         <Transportation::StudyAreaMap::StudySelectionToggler
           @selectedFeatureArray={{selectedFeatures}}
           @analysis={{analysis}} />
       </Mapbox::FeatureSelector>
+      <Mapbox::CurrentMouseoverFeatures
+        @map={{map}}
+        @layerId={{layer.layerId}}
+        @onFeatures={{this.setFirstHoveredFeatureId}}
+      />
+    </carto-source.layer>
+    <carto-source.layer
+      @id="tracts-hover"
```

Comment:
call this layer "tracts-highlight" for when it ultimately is all highlighted features?

Reply:
Oh yeah. I shall pluralize! Thx for the catch